src/main/java/com/pollra/calculator/Number.java

```diff
@@ -0,0 +1,66 @@
+package com.pollra.calculator;
+
+import java.util.Objects;
+
+public class Number {
+
+    private static final String INTEGER_MATCHES_EXPRESSION = "-?\\d+";
+    private static final int DEFAULT_VALUE = 0;
+
+    private final int value;
+
+    public Number() {
+        this(DEFAULT_VALUE);
+    }
+
+    public Number(int value) {
+        this.value = value;
+    }
+
+    public Number(String text) {
+        if( ! text.matches(INTEGER_MATCHES_EXPRESSION)) {
+            throw new RuntimeException("문자열은 숫자로 치환할 수 없습니다.");
+        }
+        if( ! text.chars()
```

Comment:
```suggestion
        if(!text.matches(INTEGER_MATCHES_EXPRESSION)) {
            throw new RuntimeException("문자열은 숫자로 치환할 수 없습니다.");
        }
        if(!text.chars()
```
개인적으론 위와 같이 선언하는것을 선호합니다 😄 
스페이스도 적절하게 사용하면 가독성을 높일수 있어요 !

Reply:
개인적으로는 전자를 더 선호합니다
이유는 마찬가지로 가독성 때문이라고 생각합니다
느낌표가 붙어있다면 다음의 코드에서 눈에 인식하기 어려울 것이라고 생각합니다
```java
if(!illigalCondition){
    ...
}
```